### PR TITLE
Conversation Editor and Questions Components

### PR DESCRIPTION
**Pull request by HerlinRijo**
Added the stakeHolder Component to the conversation editor component page. Also added the questions component to the reflection page. Components created are in their own folders so there is no confusion of what goes where. Didnt delete any of the previous conversation editor parts, just commented the lines out. Still working on styling for mobile use, havent looked into it yet, but can be on the shelf atm.

### CODE REVIEW

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/QuestionComponent/question.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      <Box
+        display="flex"
+        flexDirection="row"
+        p={1}
+        m={1}
+        bgcolor="background.paper"
+      >
+        <Box p={1}>
+          <form noValidate autoComplete="off">
+            <TextField
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/QuestionComponent/question.js:35]: Fixed this error that some other people made in my Cleanup PR, but for reference you can't have textfields inside of forms I believe.

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/StakeHoldersComponent/table.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+  return (
+    <TableContainer component={Paper}>
+      <Table className={classes.table} aria-label="simple table">
+        <TableHead>
+          <TableRow>
+            <TableCell>Issue</TableCell>
+            <TableCell align="right">Score</TableCell>
+            <TableCell align="right">Max</TableCell>
+            <TableCell align="right"></TableCell>
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/StakeHoldersComponent/table.js:44]: Why is 'max' and 'score' aligned right and 'issue' not? I would align them all to the left or align them center.

**Comment by Enoch-Hsiao** [comment]
Lot of white space in question dialog.

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/StakeHoldersComponent/stakeHolder.js [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+
+
+
+
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/StakeHoldersComponent/stakeHolder.js:134]: Needs better naming for clarification (handleClickAddQuestions)

**Comment by Enoch-Hsiao** [comment]
Would like to get this in before demo, so we could handle cleaning up code and css (dynamic width instead of static width lengths) in a later PR.

**Comment by eneadodi** [comment]
Tested on local and ran through the code. Looks, as Fisher would say, Spiffy!

**Review by eneadodi — APPROVED** [review]

**Pull request merged**